Mathlib/Analysis/NormedSpace/Ray.lean

```diff
@@ -37,11 +37,10 @@ theorem norm_add (h : SameRay ℝ x y) : ‖x + y‖ = ‖x‖ + ‖y‖ := by
 
 theorem norm_sub (h : SameRay ℝ x y) : ‖x - y‖ = |‖x‖ - ‖y‖| := by
   rcases h.exists_eq_smul with ⟨u, a, b, ha, hb, -, rfl, rfl⟩
-  wlog hab : b ≤ a with H
+  wlog hab : b ≤ a generalizing a b with H
```

Comment:
The old proof used the universe in `{F : Type _}` (which is a variable not appearing in the theorem statement) in the `wlog` generalization, to match it with that of `ℝ`! (so replacing `{F : Type _}` with `{F : Type*}` broke the proof for dark magic reasons)